pet/components/ChangePasswordDialog/index.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { TransitionProps } from '@mui/material/transitions';
+import { fira_sans_600, fira_sans_800 } from '../../core/theme/theme';
+import { Grid, Zoom, InputAdornment, IconButton } from '@mui/material';
+import { useForm } from 'react-hook-form';
+import { IChangePassword } from '../../services/api/v1/user/type';
+import { CustomPinkTextField } from '../CustomInput/type';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import Visibility from '@mui/icons-material/Visibility';
+import { useState } from 'react';
+import useToastUI from '../../core/hooks/useToastUI';
+
+const Transition = React.forwardRef(function Transition(
+    props: TransitionProps & {
+        children: React.ReactElement<any, any>;
+    },
+    ref: React.Ref<unknown>,
+) {
+    return <Zoom ref={ref} {...props} />;
+});
+
+type ConfirmDialogProps = {
```

Comment:
Suggestion(Optional): export this type in CustomDialog and import to this file would be better